### ISSUE TITLE
redirecting to Flickr Album once clicked on the gallery

### ISSUE THREAD

**Issue by thitetaf** [miromannino/Justified-Gallery#140]
Hello,

first of all congrats for this fantastic plugin. Really nicely designed, stable and professional. Have a question though, I want to avoid showing all pictures album on my blogpost, is there a way once clicking on the gallery to redirect to the Flickr Album instead of showing the image ?

thanks and regards
thibault

**Comment by miromannino** [comment]
Yes, you just have to avoid the use of lightboxes (e.g. colorbox or swipebox).
For example the [Flickr Justified Gallery for Wordpress](https://wordpress.org/plugins/flickr-justified-gallery/) does that. The thumbnail has the image itself as a src, and the link is something that points to the Flickr album. 

Let me know

**Comment by thitetaf** [comment]
Tks
Just put none in the light box section. Seems to redirect to Flickr picture
and not the album. How shall I do ? Am indeed using the Flickr Justified
Gallery

Cheers
Tibo

**Comment by thitetaf** [comment]
Hello Miro,

Did you manager to look? really struggle to redirect to the album itself

let me know
cheers
Tibo
